examples/webgl-overlay/index.js

```diff
@@ -8,33 +8,20 @@ d3.json('star-data.json', data => {
     const x2 = x.copy();
     const y2 = y.copy();
 
-    // create colour attribute buffer
-    const colors = new Float32Array(data.length * 4);
-    let i = 0;
-    data.forEach(d => {
-        colors[i++] = d.color[0];
-        colors[i++] = d.color[1];
-        colors[i++] = d.color[2];
-        colors[i++] = d.color[3];
-    });
-    const colorBuilder = fc.attributeBuilder(colors);
-
     const getWebglSeries = () =>
```

Comment:
Can we remove the `get**()` functions? I think all of the code can be inlined.

I appreciate they pre-date your involvement but can we tidy it up while we're moving it over.